### PR TITLE
Filter non-members from bulksystem messages

### PR DESCRIPTION
**Pull request by joshblum**

### CODE REVIEW

**Comment by joshblum** [comment]
(ci is a flake)

File: go/chat/uithreadloader.go [hunk 1]
```diff
@@ -119,12 +129,28 @@ func (t *UIThreadLoader) groupThreadView(ctx context.Context, uid gregor1.UID, t
 					}
 				}
 			}
+
+			// filter the usernames for people that are actually part of the team
+			for _, username := range usernames {
+				uid, err := t.G().GetUPAKLoader().LookupUID(ctx, libkb.NewNormalizedUsername(username))
```
Comment (mmaxim) [inline go/chat/uithreadloader.go]: Could just use `UIDMapper`

File: go/chat/uithreadloader.go [hunk 2]
```diff
@@ -94,9 +97,16 @@ func (t *UIThreadLoader) groupThreadView(ctx context.Context, uid gregor1.UID, t
 				Joiners: joiners,
 				Leavers: leavers,
 			})
-			return chat1.NewMessageUnboxedWithValid(mvalid)
+			msg := chat1.NewMessageUnboxedWithValid(mvalid)
+			return &msg
 		})
 
+	// make this a map once to filter BULKADDTOCONV messages
+	activeMap := make(map[string]struct{})
```
Comment (mmaxim) [inline go/chat/uithreadloader.go]: Make this on demand if we need to use it.

File: go/chat/uithreadloader.go (hunk 1 above)
Comment (joshblum) [reply go/chat/uithreadloader.go]: that goes username -> uid though

File: go/chat/uithreadloader.go [hunk 3]
```diff
@@ -412,6 +440,10 @@ func (t *UIThreadLoader) LoadNonblock(ctx context.Context, chatUI libkb.ChatUI,
 	if pagination != nil && pagination.Last {
 		return nil
 	}
+	conv, ierr := utils.GetUnverifiedConv(ctx, t.G(), uid, convID, types.InboxSourceDataSourceAll)
```
Comment (mmaxim) [inline go/chat/uithreadloader.go]: I just noticed this moved up here, let me think about this a little...

**Comment by joshblum** [comment]
@mmaxim this is ready

**Pull request merged**